docs/installation/setup.rst

```diff
@@ -16,15 +16,49 @@ difference in setup procedure.
 For this installation, 5 machines are required (or 3, if running workload
 applications on your control plane nodes).
 
-Sizing
-------
-Each machine should have at least 2 CPU cores, 4 GB of RAM, and a root
-partition larger than 40 GB.
+Machines must **not** be managed by any configuration management system
+(e.g. salt, puppet).
```

Comment:
Personnaly not sure we want to put `Salt` in this list as we may want to do it after a bootstrap (so you will have salt installed)

Reply:
For me, this documentation is to be read before doing a bootstrap or anything else, and if user already use salt to manage its machines, this can lead to unintended behaviors.